stock_warehouse_flow/models/stock_route.py

```diff
@@ -4,8 +4,8 @@
 from odoo import api, fields, models
 
 
-class StockLocationRoute(models.Model):
-    _inherit = "stock.location.route"
+class StockRoute(models.Model):
+    _inherit = "stock.route"
 
     flow_id = fields.Many2one(
```

Comment:
Since the data model has been renamed by Odoo, do you know if we are losing the `flow_id` column in the process or it's all good?
This field is set at line 363 in `_generate_delivery_route` method, and used in `_compute_impacted_route_ids` at line 187.

Reply:
I don't know, we haven't tested any migration yet as our customers are new to v16. I understand that there should be no problems.

Reply:
Okay, that was to get a feedback, thank you!